.gitmodules

```diff
@@ -1,4 +1,3 @@
 [submodule "minion-bindings/vendor"]
 	path = solvers/minion/vendor
 	url = https://github.com/niklasdewally/minion
```

Comment:
Are we able to change the submodule to minion/minion? also pointing to a specific commit hash probably?

Reply:
I still need a temporary addition in the headerfile for exposing minion_main for testing purposes.

Also I will probably be writing some c++ for minion this week, and will need to be able to depend on this before the PRs are accepted - especially as Chris might have limited / no internet next week...

This does point to a specific hash - the submodule file itself (solvers/minion/vendors) works as a reference to a specific commit.. (this can be updated by going into the submodule, pulling / checking out to a new commit , then committing to the parent-repo again.)